weather/src/js/components/WeatherApp.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Table,td,tr,th,thead,tbody } from 'react-bootstrap';
+import { getLocationCoords, getWeatherData } from '../api/weather-api';
+
+export default class WeatherApp extends React.Component {
+
+  constructor() {
+    super(); // A call to the class we are extending: React.Component
+
+    this.state = {
```

Comment:
you can use class properties instead and avoid using a `constructor`

https://babeljs.io/docs/plugins/transform-class-properties/